doc/release/1.13.0-notes.rst

```diff
@@ -159,6 +159,11 @@ being iterated over.
 For consistency with ``ndarray`` and ``broadcast``, ``d.ndim`` is a shorthand
 for ``len(d.shape)``.
 
+``np.block`` function for creating blocked arrays
+-------------------------------------------------
+Add a new ``block`` function to the current stacking functions ``vstack``,
+``hstack``, and ``stack``. It is similar to Matlab's square bracket
+notation for creating block matrices.
```

Comment:
I feel like this needs an example, I have no clue what matlabs square bracked notation is and thus what block does.